libs/openFrameworks/graphics/ofPixels.h

```diff
@@ -148,18 +148,18 @@ class ofPixels_ {
 	///
 	/// As a word of caution this reallocates memory and can be a bit
 	/// expensive if done a lot.
-	void crop(int x, int y, int width, int height);
+	void crop(size_t x, size_t y, size_t width, size_t height);
 	
 	/// \brief Crop the pixels into the ofPixels reference passed in by `toPix.
 	/// at the `x` and `y` and width the new width and height. 
 	///
 	/// As a word of caution this reallocates memory and can be a bit
 	/// expensive if done a lot.
-	void cropTo(ofPixels_<PixelType> &toPix, int x, int y, int width, int height) const;
+	void cropTo(ofPixels_<PixelType> &toPix, size_t x, size_t y, size_t width, size_t height) const;
 
 	// crop to a new width and height, this reallocates memory.
-	void rotate90(int nClockwiseRotations);
-	void rotate90To(ofPixels_<PixelType> & dst, int nClockwiseRotations) const;
+	void rotate90(size_t nClockwiseRotations);
+	void rotate90To(ofPixels_<PixelType> & dst, size_t nClockwiseRotations) const;
```

Comment:
Or at least we've always allowed it because of the while-loop / modulo ...

```
	int rotation = nClockwiseRotations;
	while (rotation < 0){
		rotation+=4;
	}
	rotation %= 4;
```